openassistant/datasets/soda_synthetic_dialogue/README.md

```diff
@@ -0,0 +1,107 @@
+    - name: validation
+      num_bytes: 187161878
+      num_examples: 146346
+  download_size: 1952926579
+  dataset_size: 1902669061
+---
+
+# Dataset Card for 🥤SODA Synthetic Dialogue
+
+## Dataset Description
+
+- **Repository:**
+  [Code](https://github.com/LAION-AI/Open-Assistant/openassistant/datasets/soda_synthetic_dialogue/)
+
+## Dataset Summary
+
+🥤SODA Synthetic Dialogue is a set of synthetic dialogues between Assistant and
+User. In each conversation, User asks Assistant to perform summarization or
+story generation tasks based on a snippet of an existing dialogue, story, or
+from a title or theme.
+
+This data was created by synthesizing the dialogues in
+[🥤Soda](https://huggingface.co/datasets/allenai/soda) and applying a set of
+templates to generate the conversation.
+
+Example:
+
+```
+User: Can you write the next few lines of dialogue for this scene:
+  Cornell: I know what you're thinking, Dontrell. I can sense your feelings and know exactly what you're going to say or do next. You're thinking about how much you miss your family, and how you wish you could see them again. You're wondering if there's anything you can do to make that happen. And you're also feeling a little guilty, because you know that if you could see them again, it would only be for a short while before they died.
+  Dontrell: Cornell, how can you possibly know all that? It's like you can read my mind!
+Assistant: Sure, the next dialogue for this scene could be:
+  Cornell: I told you, I can sense your feelings. It's like I have a connection to your thoughts and emotions. I'm not sure why or how it works, but ever since we met, I've been able to understand what you're thinking and feeling pretty easily.
+  Dontrell: Wow...that's amazing! And a little bit scary too. But it does explain why we've always been so close - even though we've only known each other for a few months.
+  Cornell: Yeah, I guess our connection is pretty strong. But don't worry, Dontrell - I'll never use it to manipulate or control you in any way. I promise."
+  Dontrell: That's a relief! I was starting to feel a little bit like you could read my mind, and it was kind of creepy. But if you promise not to use your abilities to control me, then I'm okay with it.
+  Cornell: I swear, Dontrell. I'll never use my abilities to control or manipulate you in any way. You have my word.
+User: And a title?
+Assistant: Sure, a title for this dialogue could be:
+  Cornell is intuitive. Cornell reads Dontrell's mind.
+User: How about a short description?
+Assistant: Sure, a short description for this dialogue could be:
+  Cornell knows what Dontrell is thinking. He can sense her feelings and know exactly what she is going to say or do next.
+User: What would be one theme of this story?
+Assistant: One theme of this story could be:
+  being intuitive
+```
+
+## Usage
+
+The dataset contains one configuration, `dialogue_modeling`, which has a single
+text `conversation` feature.
+
+## Source data
```

Comment:
Let's give attribution to the original dataset creators: https://arxiv.org/pdf/2212.10465v1.pdf

Reply:
The `README.md` currently has the model summary:

This data was created by synthesizing the dialogues in
[🥤Soda](https://huggingface.co/datasets/allenai/soda) and applying a set of
templates to generate the conversation.

It links to Soda's HF, but do you think we should also include a link to the paper? Not a big deal to do both imo

Reply:
I'm fine with only having the link to the dataset, so attribution is maintained, but linking the paper costs nothing, so up to you :) let me know how you decide.

Reply:
Always best to give more direct credit to the author - I've seen others be under fire for not directly giving credit to others so don't want folks here to go through that :)

Reply:
Added the paper link to the README -- should be good to go now @yk